site/troubleshooting.md

```diff
@@ -290,6 +291,27 @@ Proxy](/site/tasks/weave-docker-api/weave-docker-api.md) will re-attach any appl
 containers that it originally attached to the Weave network when they
 restart.
 
+## <a name="v2plugin"></a>Troubleshooting the V2 plugin
+
+If you have installed Weave Net via `docker plugin install`, then you
+will need to download the `weave` script to run `weave status`, `weave ps`
+or `weave report` as above.  Do this as described in the
+[install instructions](/site/install/installing-weave.md).
+
```

Comment:
(Removing some passive voice.)

If Weave Net is installed via `docker plugin install` download the `weave` script and run `weave status`, `weave ps` or `weave report`.   Install Weave Net by following the [install instructions](/site/install/installing-weave.md).